modules/020-deckhouse/monitoring/prometheus-rules/deckhouse.yaml

```diff
@@ -322,3 +322,23 @@
         Deckhouse does not work as expected.
 
         You can find out the exact problem in the relevant alerts.
+
+  - alert: D8DeckhouseConfigInvalid
+    expr: deckhouse_config_values_errors_total > 0
```

Comment:
Are you sure about this promql expression?

Per spec, `deckhouse_config_values_errors_total` is a counter, so it can only go upward.
![image](https://user-images.githubusercontent.com/32434187/146437346-5b6ead3a-00f2-437b-a0c6-f9a605f2869e.png)

Reply:
Added new metric behavior in [addon-operator](https://github.com/flant/addon-operator/commit/d86a065d5763685c6e8ed93610009eb6405ce63d) and fix it here